src/services/profiles.js

```diff
@@ -121,7 +121,7 @@ export const deleteProfile = api => async ({ user }) => {
       `,
     },
   });
-
+  
```

Comment:
Have this PR's changes been run through prettier?

Reply:
I have it set to run on save command in Sublime Text, I may missed a keystroke when saving some of the files. I think it's possible to write a git-hook so it all commits or PRs get ran through prettier isn't it?

Reply:
Yeah I think you're right